system/lib/wasmfs/backends/opfs_backend.cpp

```diff
@@ -0,0 +1,279 @@
+// Copyright 2022 The Emscripten Authors.  All rights reserved.
+// Emscripten is available under two separate licenses, the MIT license and the
+// University of Illinois/NCSA Open Source License.  Both these licenses can be
+// found in the LICENSE file.
+
+#include "backend.h"
+#include "file.h"
+#include "support.h"
+#include "thread_utils.h"
+#include "wasmfs.h"
+#include <stdlib.h>
+
+namespace wasmfs {
+
+using ProxyWorker = emscripten::ProxyWorker;
+
+extern "C" {
```

Comment:
Seems odd to put the extern "C" block inside a C++ namespace... maybe move the namespace usage below this block?